00-HTML-CSS-basics/Exercise 2.5.2.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        <title></title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="apple-touch-icon" href="apple-touch-icon.png">
+        <!-- Place favicon.ico in the root directory -->
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/grid.css">
+        <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+    </head>
+    <body>
+        <div class="container">
+            <div class="showcase"><b>Showcase</b></div>
+            <div class="info">Exercise 2.5.2. This exercise is about learning how to desing a site with css layouts. 
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer et malesuada mi. Curabitur viverra, purus 
+            non dapibus dapibus, quam mauris tincidunt erat, vel congue arcu sapien ac magna. Ut lacinia odio sit amet 
+            nisl porttitor, elementum eleifend velit laoreet. Morbi vitae metus quis nunc vehicula iaculis. Morbi nec 
+            sodales lorem. Donec venenatis venenatis mi a vehicula. In commodo posuere purus, eu hendrerit ex finibus 
+            et. Mauris lacus sem, euismod at massa vitae, mollis elementum augue.  </div>
+            <div class="pic">
+                <span class="sp">Black Globant Logo</span>
+                <img class="im" src="http://www.tiempobeta.com.ar/site/media/mod_ninja_simple_icon_menu/images/globantst.jpg" alt="Image 1">
+            </div>
+            <div class="pic">
+                <span class="sp">White Globant Logo</span>
```

Comment:
I don't like the naming on these classes. Also they're not following BEM conventions.

```
<div class="pic">
    <span class="pic--label">
    <img class="pic--image">
```

Also don't like "pic" too much, it's not that good to use shortcuts in our naming